include/helib/CModulus.h

```diff
@@ -97,6 +97,9 @@ class Cmodulus
    */
   Cmodulus(const PAlgebra& zms, long qq, long rt);
 
+  //! Copy constructor
+  Cmodulus(const Cmodulus& other) { *this = other; };
```

Comment:
Default copy constructor doesn't work here?

Reply:
This is original code. The diff thinks it isn't because I moved the order around and add some comments. Looking at src/CModulus.cpp lines [194, 196], I suspect that is the reason that in this case the copy constructor essentially uses the copy assignment op.